### DISCUSSION TITLE
Apache Apisix behind a Apache HTTPD server

### DISCUSSION THREAD

**Discussion by LuciferSam86** [apache/apisix#8063] [Q&A]
Hello there,
I would like to test an Apisix installation behind an Apache HTTPD server.

the httpd server will serve the let's encrypt certificate and then forward to apisix.
Is there some documentation to read?

Thank you!

**Comment by tzssangglass** [comment]
work for you? ref: https://apisix.apache.org/docs/apisix/certificate/
